src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -365,12 +379,19 @@ CELER_FUNCTION real_type VecgeomTrackView::find_safety(real_type max_radius)
     CELER_EXPECT(!this->is_outside());
     CELER_EXPECT(!this->is_on_boundary());
     CELER_EXPECT(max_radius > 0);
-    real_type safety = detail::BVHNavigator::ComputeSafety(
-        detail::to_vector(this->pos()), vgstate_, max_radius);
+
+    real_type safety = BVHNavigator::ComputeSafety(
+        detail::to_vector(this->pos()),
+        vgstate_
+#if VECGEOM_VERSION < 0x020000
+        ,
+        max_radius
+#endif
+    );
 
     // Since the reported "safety" is negative if we've moved slightly beyond
     // the boundary of a solid without crossing it, we must clamp to zero.
-    return max<real_type>(safety, 0);
+    return max<real_type>(0., min<real_type>(safety, max_radius));
```

Comment:
Leave this as is:
```suggestion
    return max<real_type>(safety, 0);
```
And just change the `real_type safety = BVHNavigator::ComputeSafety` call to have a `min` inside the `VECGEOM_VERSION < 0x020000` conditional.

Reply:
Your suggestion will make the `max_radius` ineffective for VecGeom v1.x.  Kept as is for now.